images/images.go

```diff
@@ -33,8 +42,13 @@ func (i Image) HasURL() bool {
 
 type Provider interface {
 	GetImage(ctx context.Context, token string) (*Image, error)
```

Comment:
Can we also document this method?

Reply:
This is going to be deleted as soon as all notifiers use the two new methods, so I thought that documenting a method that existed since this package was created right when it's gonna be deprecated didn't make much sense.
I could document it anyway if you think that's better.

Reply:
You can use golang's deprecation: https://zchee.github.io/golang-wiki/Deprecated/

```suggestion
        Deprecated: Please use either GetImageURL or GetRawImage
	GetImage(ctx context.Context, token string) (*Image, error)
```